base/applications/fontview/lang/ro-RO.rc

```diff
@@ -12,18 +13,18 @@ STRINGTABLE
 BEGIN
     IDS_INSTALL "Inst&alează"
     IDS_PRINT "I&mprimă"
-    IDS_STRING "Turubinele eoliene generează câțiva MJ (câțiva kW•h) în exces, acoperind și necesarul familiei. QY 1234567890"
-    IDS_OPEN "Deschidere font…"
+    IDS_STRING "Turbinele eoliene generează câțiva MJ (câțiva kW•h) în exces, acoperind și necesarul familiei. QY 1234567890"
+    IDS_OPEN "Deschiderea fontului…"
```

Comment:
This use of [definiteness](https://en.wikipedia.org/wiki/Definiteness) here does not look like an improvement to me.